### PR TITLE
[cleanup] erefactor/EclipseJdt - Add missing @override annotation

### PR DESCRIPTION
**Pull request by cal101**
Not sure if you care for @Override annotations and if it is useful to clean up this way? It's a lot of changes.

EclipseJdt cleanup 'AddMissingOverrideAnnotation' applied by erefactor.

For EclipseJdt see https://www.eclipse.org/eclipse/news/4.19/jdt.php
For erefactor see https://github.com/cal101/erefactor

### CODE REVIEW

File: org.eclipse.m2e.core.ui.tests/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -3,7 +3,7 @@ Fragment-Host: org.eclipse.m2e.core.ui;bundle-version="[1.16.0,2.0.0)"
 Bundle-ManifestVersion: 2
 Bundle-Name: %Bundle-Name
 Bundle-SymbolicName: org.eclipse.m2e.core.ui.tests
-Bundle-Version: 1.18.1.qualifier
+Bundle-Version: 1.18.2.qualifier
```
Comment (mickaelistria) [inline org.eclipse.m2e.core.ui.tests/META-INF/MANIFEST.MF:6]: Is this necessary? 1.18.1 was not released yet, so we can most likely stick to 1.18.1

File: org.eclipse.m2e.core.ui.tests/pom.xml [hunk 2]
```diff
@@ -23,7 +23,7 @@ Contributors:
 
   <artifactId>org.eclipse.m2e.core.ui.tests</artifactId>
   <packaging>eclipse-test-plugin</packaging>
-  <version>1.18.1-SNAPSHOT</version>
+  <version>1.18.2-SNAPSHOT</version>
```
Comment (mickaelistria) [inline org.eclipse.m2e.core.ui.tests/pom.xml:26]: same as above

**Comment by cal101** [comment]
I take a look at the version bumper.

File: org.eclipse.m2e.core.ui.tests/META-INF/MANIFEST.MF (hunk 1 above)
Comment (cal101) [reply org.eclipse.m2e.core.ui.tests/META-INF/MANIFEST.MF]: Fixed, a wrong baseline was used.

File: org.eclipse.m2e.core.ui.tests/pom.xml (hunk 2 above)
Comment (cal101) [reply org.eclipse.m2e.core.ui.tests/pom.xml]: Fixed, a wrong baseline was used.

**Comment by cal101** [comment]
@mickaelistria Anything missing here?

**Comment by mickaelistria** [comment]
Thanks!

**Pull request merged**